pontoon/base/admin.py

```diff
@@ -210,9 +212,10 @@ def enabled(self, obj):
                     "disabled",
```

Comment:
This should be moved to the very bottom of the list for consistency with the list view.